pkg/crypto/tinkcrypto/crypto.go

```diff
@@ -92,14 +106,14 @@ func (t *Crypto) Decrypt(cipher, aad, nonce []byte, kh interface{}) ([]byte, err
 		return nil, errBadKeyHandleFormat
 	}
 
-	a, err := aead.New(keyHandle)
+	ps, err := keyHandle.Primitives()
 	if err != nil {
-		return nil, fmt.Errorf("create new aead: %w", err)
+		return nil, fmt.Errorf("get primitives: %w", err)
```

Comment:
oh, I see it's the pattern of error messages here. You can probably disregard my other comment.